docs/core/tutorials/using-on-macos.md

```diff
@@ -49,95 +44,78 @@ terminal window, if you prefer).
 
 By the time we're done, you'll create three projects: a library project,
 tests for that library project, and a console application that makes
-use of the library. You'll follow a standard folder structure for
-the three projects. Following this standard folder structure
-means that the .NET Core SDK tools understand the relationship between
-your production code projects and your test code projects. That makes
-your development experience more productive.
+use of the library. 
 
 Let's start by creating those folders. In the terminal, create a 'golden'
```

Comment:
Should we mention using `dotnet sln` here to create the main folder and .sln file.  See [F# Getting Started Update](https://github.com/dotnet/docs/blob/5850b3745e7b7959f0a6013b4af6d61418da0018/docs/fsharp/tutorials/getting-started/getting-started-command-line.md#building-a-simple-multi-project-solution)